Appraisals

```diff
@@ -95,7 +95,7 @@ RAILS_VERSIONS = %w(
   4.2.5.1
   4.2.5.2
   4.2.6
-  5.0.0.beta3
+  5.0.0.1
```

Comment:
OK, don't take it as an offence, but what will this knowledge give you? I think that when you decide to test something (piece of code, method, library), you should only be testing what's necessary. Would you add tests for private methods defined in a model/class?

Anyway, I'll add entries for 4.2.7 and 5.0.0 just to close this PR and topic.